bowling.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <script src="js/jquery-3.1.0.js"></script>
```

Comment:
Usually, you would put script tags after the <body> so that they don't block the page loading.